src/components/Savings/SavingsModal/index.tsx

```diff
@@ -139,15 +146,15 @@ const SavingsModal = memo(
 
         const depositOrWithdraw = async (amount: string) => {
             if (account) {
-                sendData({ event: 'savings', action: [type] + 'Send', amount: amount })
+                sendData({ event: 'savings', action: 'savings_' + type + '_confirm', amount: amount, network })
                 const parsedAmount = (parseFloat(withdrawAmount.toFixed(0)) * 1e2).toString()
                 const tx =
                     type === 'withdraw'
                         ? await withdraw(parsedAmount)
                         : await transfer((parseFloat(amount) * 1e2).toString())
 
                 if (tx) {
-                    sendData({ event: 'savings', action: [type] + 'Success', amount: amount })
+                    sendData({ event: 'savings', action: 'savings_' + type + '_success', amount: amount, network })
```

Comment:
again, some 'success' event - I suppose should be moved after `await`

Reply:
@L03TJ3 please check other similar cases over Your changeset

Reply:
wdym after? maybe I dont understand it correctly, but it is after await no?
once we get the tx back, it means its successfully send to blockchain